src/cli.js

```diff
@@ -1,6 +1,20 @@
 import * as jsApi from './node-jspm-jasmine.js';
+import commander from 'commander';
 
-jsApi.runTests({}, function(err) {
-	console.error(err);
-	process.exit(1);
-});
+export function run(args) {
+
+    commander
+        .option('-c, --jasmineConfig <path>')
+        .option('-c, --packageJSON <path>')
```

Comment:
I like this option, but should it maybe renamed to `--package-path`, since [it's supposed to be a directory](https://github.com/jspm/jspm-cli/blob/master/docs/api.md#setpackagepathpackagepath)? At least that's what I understand from JSPM's docs.